### ISSUE TITLE
Using Multiprocessing on Bruker Locally

### ISSUE THREAD

**Issue by jmdelahanty** [Tyelab/bruker_pipeline#4]
We can execute the ripper using Python and multiprocessing locally on Bruker without any of the slowdowns or issues currently being experienced on the remote methods as described in #1 and #3. These issues should be discussed further with @laurelrr when time permits. To reiterate the issues here briefly with Beyblade, it works efficiently/as well as the local machine until it suddenly gets mysteriously bogged down by something after around 20k images. The Docker container for some reason is not accessing the `lxml` library when it needs to given how `Prairie View` outputs malformed `.xml` files when using the stimulation functionality. Until that is solved, development for using Cheetos is stuck as I've been hacking away for a bit but have been unsuccessful.

Additional testing should be worked out for simply performing the ripping serially for the day's recordings from multiprocessing with a single process, using tifffile with zarr/h5py for concatenation,  and then using `shutil`  for automatically transferring the files to the server at the end of the day as well as deleting the intermediate OME.tif files. Once this is worked out, a new issue for performing the procedure in parallel will be created on this repo.
